servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -110,8 +110,10 @@ quarkus.resteasy.gzip.enabled=true
 #quarkus.oidc.auth-server-url=
 # fixed at buildtime
 quarkus.http.auth.basic=false
-quarkus.oidc.enabled=false
-
+quarkus.oidc.enabled=true
+nessie.auth.enabled=false
```

Comment:
maybe rename this to `nessie.server.authentication.enabled` to be consistent with other settings that we have

Reply:
will do

Reply:
Renamed to `nessie.server.auth.enabled` (for brevity) in 9afbec6a279d242e5565ed38bd53523f6ca9ab67